samples/tab-graph-toolkit/nodejs/teamsapp.local.yml

```diff
@@ -0,0 +1,85 @@
+# yaml-language-server: $schema=https://aka.ms/teams-toolkit/1.0.0/yaml.schema.json
```

Comment:
this project need an AAD App. Will this work?

Reply:
Yes tested this application
![image](https://github.com/OfficeDev/Microsoft-Teams-Samples/assets/109147706/3af3c97c-44e4-40bb-87db-7778c5fb9875)

Reply:
Yes, tested this sample.
![image](https://github.com/OfficeDev/Microsoft-Teams-Samples/assets/109147706/d9760d00-e120-40ab-8b99-77060e86d03f)